tests/ssl-opt.sh

```diff
@@ -1563,6 +1563,20 @@ run_test    "TLS 1.3, TLS_AES_128_CCM_8_SHA256, ext PSK, early data" \
 	    -s "<= parse early data"                        \
 	    -s "<= parse end_of_early_data"
 
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
+requires_config_enabled MBEDTLS_DEBUG_C
+requires_config_disabled MBEDTLS_RSA_C
+requires_config_enabled MBEDTLS_ZERO_RTT
+run_test    "TLS 1.3, TLS_AES_256_GCM_SHA384, ECDHE-ECDSA, SRV disables early data, client enables early data" \
+            "$P_SRV nbio=2 debug_level=5 force_version=tls1_3" \
+            "$P_CLI nbio=2 debug_level=5 force_version=tls1_3 force_ciphersuite=TLS_AES_256_GCM_SHA384 early_data=1" \
+            0 \
+      -s "Protocol is TLSv1.3"                                        \
+      -c "<= skip write early_data extension"                         \
+      -c "Protocol is TLSv1.3"                                        \
+      -c "Ciphersuite is TLS_AES_256_GCM_SHA384"                      \
+      -c "early data status = 0"
```

Comment:
Does this indicate the client never send early data?

Reply:
Yes, the client shouldn't as there is no external PSK in the first request.

Reply:
So what are we testing here? Should we change the description to 'client enable early data but no PSK'?